src/main/scala/newmodel/DocElement.scala

```diff
@@ -0,0 +1,75 @@
+package newmodel
```

Comment:
Actually, @MasseGuillaume what do you think is a good way to progress and allow you to re-use this work in metadoc? Building out own tree? Or copying meta's and replacing components? Any other suggestions?